### PR TITLE
docs(fix): wording

### PR DESCRIPTION
**Pull request by bene2k1**

### CODE REVIEW

File: compute/bare-metal-cloud/how-to/order-flexible-ip.mdx [hunk 1]
```diff
@@ -11,7 +11,7 @@ dates:
   posted: 2021-05-26
 ---
 
-[Flexible IP addresses](/compute/bare-metal-cloud/concepts/#flexible-ip) are additional IP addresses, available for [Bare Metal Cloud Servers](https://www.scaleway.com/en/bare-metal-servers/). They allow you to move an IP from one server to another without changing your whole configuration. Flexible IPs can also be used as additional IP addresses to create virtual machines on your Bare Metal Cloud Server.
+[flexible IP addresses](/compute/bare-metal-cloud/concepts/#flexible-ip) are additional IP addresses, available for [Bare Metal Cloud Servers](https://www.scaleway.com/en/bare-metal-servers/). They allow you to move an IP from one server to another without changing your whole configuration. Flexible IPs can also be used as additional IP addresses to create virtual machines on your Bare Metal Cloud Server.
```
Comment (RoRoJ) [inline compute/bare-metal-cloud/how-to/order-flexible-ip.mdx:14]: Is this the start of a sentence though, should it still be capitalised?

File: compute/instances/concepts.mdx [hunk 2]
```diff
@@ -82,7 +82,7 @@ Cloud-init is a multi-distribution package that [provides boot time customizatio
 
 ## Flexible IP
 
-Flexible IP addresses are public IP addresses that you can hold independently of any Instance. When you create a Scaleway Instance, its public IP address is also a flexible IP address. Flexible IP addresses can be attached to and detached from any of your Instances as you wish. You can keep a number of flexible IP addresses in your account at any given time. When you delete a flexible IP address, it is disassociated from your account to be used by other users. Find out more: [how to use flexible IP addresses](/compute/instances/how-to/use-flexips/)
+flexible IP addresses are public IP addresses that you can hold independently of any Instance. When you create a Scaleway Instance, its public IP address is also a flexible IP address. flexible IP addresses can be attached to and detached from any of your Instances as you wish. You can keep a number of flexible IP addresses in your account at any given time. When you delete a flexible IP address, it is disassociated from your account to be used by other users. Find out more: [how to use flexible IP addresses](/compute/instances/how-to/use-flexips/)
```
Comment (RoRoJ) [inline compute/instances/concepts.mdx:85]: Start of a sentence?

File: dedibox-network/ip-failover/how-to/index.mdx [hunk 3]
```diff
@@ -1,4 +1,114 @@
 ---
```
Comment (RoRoJ) [inline dedibox-network/ip-failover/how-to/index.mdx:1]: I think this must be a copy/paste error?

File: faq/billing.mdx [hunk 4]
```diff
@@ -34,7 +34,7 @@ Below is a detailed list of what is considered as start and stop billing events
...
     | Apple silicon M1 as-a-Service  |         The moment the Mac mini is **created**.                | The moment the Mac mini is **deleted**. | Due to license constraints, the minimum lease for Apple silicon-as-a-Service is 24 hours. As a result, **a Mac mini M1 can only be deleted after a minimum allocation period of 24 hours**.<br /><br />When a Mac mini M1 is powered off, the physical machine remains allocated to the user’s account, meaning the resource continues to be billed. |
-    | Flexible IP Address |     The moment you reserve an IP.                  |     The moment you cancel a reservation.                       | This class of IP is billed as long as you keep the reservation open. You will be billed whether you are effectively using the IP or not. |
+    | flexible IP Address |     The moment you reserve an IP.                  |     The moment you cancel a reservation.                       | This class of IP is billed as long as you keep the reservation open. You will be billed whether you are effectively using the IP or not. |
```
Comment (RoRoJ) [inline faq/billing.mdx:37]: Start of a sentence, cap

File: compute/bare-metal-cloud/concepts.mdx [hunk 5]
```diff
@@ -40,7 +40,7 @@ Bare Metal servers allow you to order dedicated servers on-demand, like cloud In
 
 ## Flexible IP
 
-Flexible IP addresses are public IP addresses which you can hold independently of any Instance. When you create a Scaleway Instance, its public IP address is also a flexible IP address. Flexible IP addresses can be attached to and detached from any of your Instances as you wish. You can keep a number of flexible IP addresses in your account at any given time. When you delete a flexible IP address, it is disassociated from your account to be used by other users.
+flexible IP addresses are public IP addresses which you can hold independently of any Instance. When you create a Scaleway Instance, its public IP address is also a flexible IP address. flexible IP addresses can be attached to and detached from any of your Instances as you wish. You can keep a number of flexible IP addresses in your account at any given time. When you delete a flexible IP address, it is disassociated from your account to be used by other users.
```
Comment (RoRoJ) [inline compute/bare-metal-cloud/concepts.mdx:43]: Start of a sentence, I think it should retain its cap

File: network/vpc/concepts.mdx [hunk 6]
```diff
@@ -47,7 +47,7 @@ The **D**omain **N**ame **S**ystem (DNS) is a naming system for devices connecte
...
+flexible IP addresses are [public IP addresses](#public-ip-address) associated with your account, which you can hold independently of any Instance or public gateway. When you create an Instance or a public gateway, it receives a flexible (public) IP address by default, which is added to your account's pool of flexible IPs. You can [detach, reattach and migrate](/network/vpc/how-to/use-flexible-ips/) your flexible IPs between your Instances and public gateways at your convenience. Note however that each Public Gateway must have a public IP attached to it. If you detach one flexible IP you must attach another. When you delete a flexible IP address, it is disassociated from your account to be used by other users.
```
Comment (RoRoJ) [inline network/vpc/concepts.mdx:50]: Start of a sentence

File: tutorials/create-esxi-cluster-dedibox/configure-centos.mdx [hunk 7]
```diff
@@ -0,0 +1,98 @@
+---
```
Comment (RoRoJ) [inline tutorials/create-esxi-cluster-dedibox/configure-centos.mdx:1]: Is this doc meant to be in this PR?

File: tutorials/pihole-vpn/index.mdx [hunk 8]
```diff
@@ -209,16 +209,16 @@ To avoid keeping an open DNS resolver on the Intenet, we restrict the requests f
...
   <Lightbox src="scaleway-create_sg.png" alt="" />
 
-4. Enter the details for your new Security Group:
+4. Enter the details for your new security group:
 
-    - **Security Group Name:** a friendly name for your Security Group, (e.g. `block-remote-dns`)
-    - **Description:** An optional description for your Security Group
-    - **Available Zone:** Choose the geographic region in which your Security Group will be deployed. The region must match the region of your Virtual Instance.
-    - **Rules:** Configure rules in your Security Group to block incoming traffic on Port 53 (DNS) to block external requests to your PiHole instance:  
+    - **security group Name:** a friendly name for your security group, (e.g. `block-remote-dns`)
```
Comment (RoRoJ) [inline tutorials/pihole-vpn/index.mdx:218]: Start of a sentence, keep cap

File: compute/bare-metal-cloud/how-to/order-flexible-ip.mdx (hunk 1 above)
Comment (bene2k1) [reply compute/bare-metal-cloud/how-to/order-flexible-ip.mdx:14]:
```suggestion
[Flexible IP addresses](/compute/bare-metal-cloud/concepts/#flexible-ip) are additional IP addresses, available for [Bare Metal Cloud Servers](https://www.scaleway.com/en/bare-metal-servers/). They allow you to move an IP from one server to another without changing your whole configuration. Flexible IPs can also be used as additional IP addresses to create virtual machines on your Bare Metal Cloud Server.
```

File: compute/instances/concepts.mdx (hunk 2 above)
Comment (bene2k1) [reply compute/instances/concepts.mdx:85]:
```suggestion
Flexible IP addresses are public IP addresses that you can hold independently of any Instance. When you create a Scaleway Instance, its public IP address is also a flexible IP address. Flexible IP addresses can be attached to and detached from any of your Instances as you wish. You can keep a number of flexible IP addresses in your account at any given time. When you delete a flexible IP address, it is disassociated from your account to be used by other users. Find out more: [how to use flexible IP addresses](/compute/instances/how-to/use-flexips/)
```

File: faq/billing.mdx (hunk 4 above)
Comment (bene2k1) [reply faq/billing.mdx:37]:
```suggestion
    | Flexible IP Address |     The moment you reserve an IP.                  |     The moment you cancel a reservation.                       | This class of IP is billed as long as you keep the reservation open. You will be billed whether you are effectively using the IP or not. |
```

File: network/vpc/concepts.mdx (hunk 6 above)
Comment (bene2k1) [reply network/vpc/concepts.mdx:50]:
```suggestion
Flexible IP addresses are [public IP addresses](#public-ip-address) associated with your account, which you can hold independently of any Instance or public gateway. When you create an Instance or a public gateway, it receives a flexible (public) IP address by default, which is added to your account's pool of flexible IPs. You can [detach, reattach and migrate](/network/vpc/how-to/use-flexible-ips/) your flexible IPs between your Instances and public gateways at your convenience. Note however that each Public Gateway must have a public IP attached to it. If you detach one flexible IP you must attach another. When you delete a flexible IP address, it is disassociated from your account to be used by other users.
```

File: compute/bare-metal-cloud/concepts.mdx (hunk 5 above)
Comment (bene2k1) [reply compute/bare-metal-cloud/concepts.mdx:43]:
```suggestion
Flexible IP addresses are public IP addresses that you can hold independently of any Instance. When you create a Scaleway Instance, its public IP address is also a flexible IP address. Flexible IP addresses can be attached to and detached from any of your Instances as you wish. You can keep a number of flexible IP addresses in your account at any given time. When you delete a flexible IP address, it is disassociated from your account to be used by other users.
```

File: tutorials/pihole-vpn/index.mdx (hunk 8 above)
Comment (bene2k1) [reply tutorials/pihole-vpn/index.mdx:218]:
```suggestion
    - **Security group Name:** a friendly name for your security group, (e.g. `block-remote-dns`)
```

File: tutorials/create-esxi-cluster-dedibox/configure-centos.mdx (hunk 7 above)
Comment (bene2k1) [reply tutorials/create-esxi-cluster-dedibox/configure-centos.mdx:1]: It is there because of "failover IP" - I don't know why it is entirely green

**Review by RoRoJ — APPROVED** [review]

File: compute/instances/how-to/use-security-groups.mdx [hunk 9]
```diff
@@ -11,7 +11,7 @@ dates:
   posted: 2021-05-26
 ---
 
-[Security Groups](/compute/instances/concepts#security-group) act as firewalls, filtering traffic on your Instances. They can be stateful or stateless, and allow you to create rules to drop or allow traffic to and from your Instance. On this page, we show you how to create, edit and configure a Security Group via the Scaleway console.
+[Security Groups](/compute/instances/concepts#security-group) act as firewalls, filtering traffic on your Instances. They can be stateful or stateless, and allow you to create rules to drop or allow traffic to and from your Instance. On this page, we show you how to create, edit and configure a security group via the Scaleway console.
```
Comment (ldecarvalho-doc) [inline compute/instances/how-to/use-security-groups.mdx:14]:
```suggestion
[Security groups](/compute/instances/concepts#security-group) act as firewalls, filtering traffic on your Instances. They can be stateful or stateless, and allow you to create rules to drop or allow traffic to and from your Instance. On this page, we show you how to create, edit and configure a security group via the Scaleway console.
```

**Review by ldecarvalho-doc — APPROVED** [review]

**Pull request merged**